src/Rules/AbstractSpacingRule.php

```diff
@@ -11,6 +11,17 @@
  */
 abstract class AbstractSpacingRule extends AbstractFixableRule
 {
+    public function __construct(private bool $skipIfNewLine = true)
+    {
+    }
+
+    public function getConfiguration(): array
+    {
+        return [
+            'skipIfNewLine' => $this->skipIfNewLine,
+        ];
+    }
```

Comment:
We can simplify to

```suggestion
    protected bool $skipIfNewLine = false;
```
And setting the value in the construct of DelimiterSpacingRule with the implementation of getConfiguration